Makefile

```diff
@@ -20,11 +20,20 @@ iso_creation: init
 .PHONY: centos_iso
 centos_iso: KICKSTART_FILE=centos-7-minimal.ks
 centos_iso: KICKSTART_TEMPLATE=centos-7-minimal.template
-centos_iso: ISO_NAME=minishift-centos.iso
+centos_iso: ISO_NAME=minishift-centos
 centos_iso: iso_creation
 
 .PHONY: rhel_iso
+ifndef rhel_tree_url
```

Comment:
+1, let me test this once I get a better connection again and then we should be go